src/reducers/copy.js

```diff
@@ -39,50 +40,71 @@ export default (pattern) => (base, source) => {
       sourceFields.forEach(f => base.insertField(f));
       return base;
     }
-    const filterMissing = function(sourceField) {
-      // Test 02: Identical control fields are not copied
-      if ('value' in sourceField) {
-        return baseFields.some(isIdenticalControlField) === false;
-      }
-      // Test 04: Identical data fields in base and source, not copied
-      // Test 05: Different data fields are copied from source to base
-      if ('subfields' in sourceField) {
-        return baseFields.some(isIdenticalDataField) === false;
-      }
 
-      function normalizeControlField(field) {
-        return field.value.toLowerCase().replace(/\s+/u, '');
-      }
+    const [baseField] = baseFields;
+    debug(`baseField: ${JSON.stringify(baseField, undefined, 2)}`);
+    const [sourceField] = sourceFields;
+    debug(`sourceField: ${JSON.stringify(sourceField, undefined, 2)}`);
+    debug(`compareTagsOnly: ${compareTagsOnly}`);
+    // Test 06: Compare tags to see if field is non-repeatable and don't copy, even if it is different
+    if (compareTagsOnly === false) {
```

Comment:
`compareTagsOnly` is much simpler & shorter so it would be better to do the implementation this way `if (compareTagsOnly) {`